lxd/device/unix_hotplug.go

```diff
@@ -91,6 +99,10 @@ func (d *unixHotplug) validateConfig(instConf instance.ConfigReader) error {
 		return fmt.Errorf("Unix hotplug devices require a vendorid or a productid")
 	}
 
+	if d.config["gid"] != "" && d.config["uid"] != "" && shared.IsTrue(d.config["ownership.inherit"]) {
+		return fmt.Errorf("Unix hotplug device ownership cannot be inherited from host while GID and UID are set")
+	}
```

Comment:
Shouldn't this error be return whenever one of `gid` or `uid` is set while `ownership.inherit == true`?

Reply:
We're allowing one of `gid` or `uid` to be set with `ownership.inherit` set to `true.

Reply:
See https://github.com/canonical/lxd/pull/14417#discussion_r1846459783.